src/foam/u2/view/ReadReferenceView.js

```diff
@@ -28,17 +92,41 @@ foam.CLASS({
       expression: function(obj) { return obj.cls_; }
     },
     'prop',
+    {
+      class: 'String',
+      name: 'linkTo',
```

Comment:
feel like this would be safer as a boolean. Consider property class boolean with name showDaoSummary

Reply:
linkTo is a pointer to the link reference (e.g., dao summary, or some menu). I am not sure how I can make this boolean.

Reply:
'linkTo' is a pointer to the link reference, and this could point to one of the menus in `menuKeys`. I am not completely sure how we can use boolean to achieve this.